core-ui/src/components/HelmReleases/HelmReleaseStatus.js

```diff
@@ -2,9 +2,8 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { StatusBadge } from 'react-shared';
 
-export function HelmReleaseStatus({ release }) {
+export function HelmReleaseStatus({ status }) {
   const { i18n } = useTranslation();
-  const status = release.info.status;
 
   const type = status === 'deployed' ? 'success' : 'info';
```

Comment:
I know it's not part of this task, but I think we should add a follow up for the release statuses since we have more than just deployed (https://github.com/helm/helm/blob/main/pkg/release/status.go)